src/main/groovy/com/github/lburgazzoli/gradle/plugin/karaf/features/KarafFeaturesTask.groovy

```diff
@@ -80,10 +80,18 @@ class KarafFeaturesTask extends DefaultTask implements KarafTaskTrait  {
 
                     feature.configFiles.each {
                         if (it.filename && it.uri) {
-                            builder.configfile(
-                                [ finalname: it.filename ],
-                                it.uri
-                            )
+                            if (it.override) {
+                                builder.configfile(
```

Comment:
Yes that is a much better solution. Sorry it took to long for me to get back to you, but thank you for implementing and incorporating these changes!